### DISCUSSION TITLE
How to execute function using synchronously？

### DISCUSSION THREAD

**Discussion by prog-ape** [tdlight-team/tdlight-java#138] [Q&A]
How to use synchronous execution function, because some operations do not need to use asynchronous

**Comment by cavallium** [comment]
For example?

**Comment by prog-ape** [comment]
For example, on the web page, view user details, etc.

**Comment by cavallium** [comment]
They are asynchronous functions in tdlib

**Comment by yvasyliev** [comment]
Hi @alon-sys, you can use an example from #12 to execute functions synchronously
